resources/about.html

```diff
@@ -0,0 +1,97 @@
+    <script src="about.js"></script>
+
+</head>
+<body>
+
+<nav>
+    <div class="nav-wrapper lime">
+        <div class="brand-logo center" style="height: auto; width: auto;">
+            <a>Octopeer</a>
+        </div>
+
+        <ul class="right hide-on-small-only">
+            <li><a href="../analytics/index.html">Analytics</a></li>
+            <li><a href="options.html">Options</a></li>
+            <li class="active"><a href="#">About</a></li>
+        </ul>
+
+        <ul id="slide-out" class="side-nav">
+            <li><a href="../analytics/index.html">Analytics</a></li>
+            <li><a href="options.html">Options</a></li>
+            <li class="active"><a href="#">About</a></li>
+        </ul>
+        <a href="#" data-activates="slide-out" class="right button-collapse hide-on-med-and-up"><i class="material-icons">menu</i></a>
+
+    </div>
+</nav>
+
+<div class="container">
+    <p class="flow-text" style="text-align: center;">
+        This page tells you all about Octopeer.
+    </p>
+
+    <div class="row">
+
+        <div class="col s12 m6">
+            <p class="flow-text">What is Octopeer?</p>
+            <p>
+                Octopeer is a chrome extension that will help you analyze your personal code reviewing behaviour.
+                While reviewing code in pull requests on both GitHub and BitBucket this extension will collect data.
+                This data can be viewed on the analytics section of our extension.
+                It gives you several useful overviews of your behaviour.
+                Doing this might make you see that you are able to improve your code reviewing behaviour.
+
+            </p>
+        </div>
+
+        <div class="col s12 m6">
+            <p class="flow-text">Why should i use Octopeer? </p>
+            <p>
+                Octopeer might make you realize things about your code reviewing behaviour you never thought off.
+                These insights in your personal behaviour will affect the way you review code.
+                The subtle changes you might make in your personal style of code review are valuable.
+                It could boost the quality of your code, or make you spend less time on doing reviews while keeping
+                the quality high.
+                Many good things might come from the use of our extension.
+                Try it out and you will see.
+            </p>
+        </div>
+
+        <div class="col s12 m6">
+            <p class="flow-text">What will happen with my data? </p>
+            <p>
+                First of all, your data will be viewed to yourself.
+                That is the main purpose of our extension, giving you personal insights.
+                Next to that the data might be used for research at the Delft University of Technology.
+                These researches can yield surprising results when the researchers are equipped with a nice dataset.
```

Comment:
Maybe mention that the way Pull Requests are reviewed might be changed because of their research?